packages/react-core/src/components/DataList/DataList.tsx

```diff
@@ -308,13 +313,12 @@ export class DataList extends React.Component<DataListProps, DataListState> {
     } = this.props;
     const { dragging } = this.state;
     const isSelectable = onSelectDataListItem !== undefined;
-    const isDraggable = onDragFinish !== undefined;
```

Comment:
So here, why don't we still need to check for onDragFinish.  I think this is needed to make it draggable.  I assumed you were not checking because you were planning on no longer supporting it.  If the drag and drop still woks when not defined, this should be ok.

Reply:
Oh, instead of checking if `onDragFinish !== undefined` in the render I now check `Boolean(props.onDragFinish || props.onDragStart || props.onDragMove || props.onDragCancel)` in the constructor. This is more permissive.

Reply:
Great! They way I am understanding the original code thought is that onDragFinished is the only one of drag props required.  since it is the only one checked.  I can play around with it and see what happens if I don't supply it.